lib/file_path_parser.go

```
@@ -0,0 +1,48 @@
+package lib
+
+import (
+	"go/build"
+	"io/ioutil"
+	"path/filepath"
+	"strings"
+)
+
+func fileFileForTest(suite *Suite, test *Test, pkg *build.Package, strip string) string {
+	var (
+		testFileContents = make(map[string]string)
+		body             []byte
+		testFilePath     string
+		contents         string
+		err              error
+		ok               bool
+	)
+	if strip == "" {
+		strip = pkg.SrcRoot + "/"
+	}
+	for _, testFile := range pkg.TestGoFiles {
+		testFilePath = filepath.Join(pkg.Dir, testFile)
+		if contents, ok = testFileContents[testFilePath]; !ok {
+			if body, err = ioutil.ReadFile(testFilePath); err != nil {
```

Don't ignore errors, unless there's a good reason